tests/StackExchange.Redis.Tests/SortedSets.cs

```diff
@@ -639,5 +639,95 @@ public void SortedSetRangeStoreFailExclude()
             var exception = Assert.Throws<ArgumentException>(()=>db.SortedSetRangeAndStore(sourceKey, destinationKey,0,-1, exclude: Exclude.Both));
             Assert.Equal("exclude", exception.ParamName);
         }
+
+        [Fact]
+        public void SortedSetScoreSingle()
+        {
+            using var conn = Create();
+            Skip.IfBelow(conn, RedisFeatures.v2_1_0);
+
+            var db = conn.GetDatabase();
+            var me = Me();
+
+            var setKey = $"{me}:ZScoreSet";
+            var memberName = $"{me}:ZScoreMember";
```

Comment:
Overall requests on these tests:
- Only the set key needs to be unique, no need to get fancier or more expensive on the members.
- We should delete the set key(s) before starting, to ensure a good test
- Should check the "set doesn't exist" (random key) case
- Also need to test the Async versions :)

Reply:
Resolved.